Adyen/Model/Modification/CaptureRequest.cs

```diff
@@ -43,6 +41,8 @@ public CaptureRequest()
         [DataMember(Name = "modificationAmount", EmitDefaultValue = false)]
         public Amount ModificationAmount { get; set; }
 
+        [DataMember(Name = "splits", EmitDefaultValue = false)]
+        public List<Split> Splits { get; set; }

```

Comment:
on API explorer I see this field also on cancel/refund requests, can you please double check if it's a mistake there or if it's needed to cancel/refund?